modules/images-other-jenkins-agent-memory.adoc

```diff
@@ -7,6 +7,7 @@
 = Jenkins agent memory requirements
 
 A JVM is used in all Jenkins agents to host the Jenkins JNLP agent as well as to run any Java applications such as `javac`, Maven, or Gradle.
+// ^ impacted?
```

Comment:
this is not impacted. It is still a valid statement and a possible hack that any user would be interested to reduce some pipeline footprints.